tests/chart_tests/test_probes.py

```diff
@@ -0,0 +1,32 @@
+import pytest
+
+import tests.chart_tests as chart_tests
+from tests import supported_k8s_versions
+
+ignore_kind_list = ["DaemonSet", "StatefulSet", "StatefulSet", "Job"]
```

Comment:
@danielhoherd Currently we are listing containers from `DaemonSet, StatefulSet, StatefulSet, Job and Deployment`.

Reply:
We have many test cases failing for kind `Deployment` also as either `liveness` or `readiness` probes are missing.

Reply:
Yeah, I noticed that while I was troubleshooting the flaky tests. I wonder how the lifecycle on these components is being managed. This might just be a huge hole for us where we need to do a bunch of work to get to a good state.